src/SolutionTemplate/Uno.ProjectTemplates.Dotnet/content/unoapp-uwp-net6/.vsconfig

```diff
@@ -3,92 +3,36 @@
   "components": [
     "Microsoft.VisualStudio.Component.CoreEditor",
     "Microsoft.VisualStudio.Workload.CoreEditor",
-    "Microsoft.NetCore.Component.Runtime.3.1",
```

Comment:
I didn't change anything in `.vsconfig`. These were done automatically when I built from VS.